tests/chainer_tests/functions_tests/math_tests/test_basic_math.py

```diff
@@ -1,7 +1,8 @@
+import operator
+import sys
 import unittest
 
 import numpy
-import six
```

Comment:
`six.assertRaisesRegex` is used in the code. Removing this line makes gpu tests fail.